index.js

```diff
@@ -32,13 +32,16 @@ function setupBilling(config) {
 
     const models = require("./models")(config.db_connection, config.collection_name, config.orm_type);
     const { getActivePlans, subscribePlan, getActiveSubscription, updateSubscriptionStatus } = require("./controllers/subscription.helper")(config, models);
+    const { WebhookHandler } = require("./handlers/webhook-handler");
+    const webhookHandler = (models) => new WebhookHandler(models);
```

Comment:
why we need function here? why can't we just create new instance directly? Think properly before adding extra unnecessary steps.